### PR TITLE
#1386 KYC - Add maximum value validator to beneficial ownership percentage

### PR DESCRIPTION
**Pull request by drakkein**

### CODE REVIEW

File: app/lib/api/KycApi.interfaces.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export interface IKycBeneficialOwner extends IKycPerson {
 }
 export const KycBeneficialOwnerSchema = KycPersonSchema.concat(
   Yup.object().shape({
-    ownership: Yup.number(),
+    ownership: Yup.number().max(100),
```
Comment (Moejoe90) [inline app/lib/api/KycApi.interfaces.ts]: lets make it a constant on top of the file with a clear name

File: app/lib/api/KycApi.interfaces.ts [hunk 2]
```diff
@@ -7,6 +7,8 @@ import {
   restrictedCountry,
 } from "./util/schemaHelpers";
 
+const percentageLimit = 100;
```
Comment (Moejoe90) [inline app/lib/api/KycApi.interfaces.ts]: full upper case please :D

Comment (drakkein) [reply app/lib/api/KycApi.interfaces.ts]: It supposed to be ez 😆

File: app/lib/api/KycApi.interfaces.ts [hunk 3]
```diff
@@ -87,9 +89,10 @@ export interface IKycBeneficialOwner extends IKycPerson {
   ownership?: number;
   id?: string;
 }
+
 export const KycBeneficialOwnerSchema = KycPersonSchema.concat(
   Yup.object().shape({
-    ownership: Yup.number(),
+    ownership: Yup.number().max(PERCENTAGE_LIMIT),
```
Comment (desfero) [inline app/lib/api/KycApi.interfaces.ts]: we should export percentage as a custom schema time and use everywhere we need to validate percentage, as here we should also set `min` to `0` as it's not possible to have a negative percentage value

Comment (desfero) [reply app/lib/api/KycApi.interfaces.ts]: Refer to implementation of https://github.com/Neufund/platform-frontend/blob/cae3d0ab274295a646e5e973e1c1de1fb62fc658/app/lib/api/eto/EtoApi.interfaces.tsx#L28

**Pull request merged**